.github/workflows/on-pullrequest.yml

```diff
@@ -0,0 +1,75 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: On Pull Request
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      # TODO: restore this with AA-1601, except run in a PS-specific
+      # workflow, so that the C# build and CodeQL do not run
+      # unnecessarily.
+      # - "**/*.ps1"
+      # - "**/*.psm1"
```

Comment:
Please remove the TODO comment and uncomment the powershell lines.

Reply:
It sounds like we are addressing this in a future ticket